tests/ha/test_data_pod_failure.py

```diff
@@ -1371,174 +1321,167 @@ def test_reads_deletes_during_pod_down(self):
         LOGGER.info("Step 1: Performing READs and DELETEs with variable object sizes during data "
                     "pod down by delete deployment.")
 
-        LOGGER.info("Perform WRITEs on %s buckets", wr_bucket)
+        LOGGER.info("Perform WRITEs on %s buckets for background DELETEs", del_total_bkt)
         args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
-                'skipget': True, 'skipdel': True, 'bkts_to_wr': wr_bucket, 'output': wr_output}
-
+                'skipget': True, 'skipdel': True, 'bkts_to_wr': del_total_bkt, 'output': wr_output}
         self.ha_obj.put_get_delete(event, s3_test_obj, **args)
         wr_resp = tuple()
         while len(wr_resp) != 3:
             wr_resp = wr_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
-        if not wr_resp:
-            assert_utils.assert_true(False, "Failed to do writes")
-        s3_data = wr_resp[0]  # Contains s3 data for passed buckets
-        event_put_bkt = wr_resp[1]  # Contains buckets when event was set
-        fail_put_bkt = wr_resp[2]  # Contains buckets which failed when event was clear
-        assert_utils.assert_false(len(fail_put_bkt) or len(event_put_bkt),
-                                  "Expected pass, buckets which failed in create/put operations "
-                                  f"{fail_put_bkt} and {event_put_bkt}.")
+        s3_data_del = wr_resp[0]  # Contains IAM user data for passed buckets
+        written_bck = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_equal(len(written_bck), del_total_bkt,
+                                  f"Failed to create {del_total_bkt} number of buckets."
+                                  f"Created {len(written_bck)} number of buckets")
+        LOGGER.info("Performed WRITEs on %s buckets for background DELETEs", del_total_bkt)
```

Comment:
Are you taking two different users for read and delete operations?
Because there will be conflict if delete process is faster than read and it tries to read deleted object. Please make sure two different users are used for these processes.

Reply:
This is specific to delete and it will delete only those buckets which is written for DELETE operation. Why it go for READs conflicts ?